packages/twenty-front/src/modules/settings/developers/webhook/utils/fetchGraphData.ts

```diff
@@ -0,0 +1,95 @@
+import { NivoLineInput } from '@/settings/developers/webhook/components/SettingsDevelopersWebhookUsageGraph';
+import { WEBHOOK_GRAPH_API_OPTIONS_MAP } from '@/settings/developers/webhook/constants/WebhookGraphAPIOptionsMap';
+import { SnackBarVariant } from '@/ui/feedback/snack-bar-manager/components/SnackBar';
+
+type fetchGraphDataProps = {
```

Comment:
feels a bit strange to have `webhookId` and `webhookOptions = 7D...` next to another. WebhookOptions feels very generic so I would have expected an object. 
Maybe you can just rename that param to `windowLength` or something like that?